segmentation/utils/visualize.py

```diff
@@ -18,3 +18,84 @@ def visualize_z_scores(z, pheno_titles):
     zplot = pd.DataFrame(z, columns=pheno_titles, index=pheno_titles)
     sns.set(font_scale=.7)
     sns.clustermap(zplot, figsize=(8, 8), cmap="vlag")
+    
+def newDict(names):
+    result = {}
+    for name in names:
+        result[name] = [0]
+    return result
```

Comment:
It's good that you're defining functions for specific tasks, but this might be a bit overkill. This is a pretty standard operation in Python, and it can be simplified into one line as well: result = {name: [0] for name in names}.

It's also good to choose a better name than "name" because it's too similar to "names" which you are iterating over.